include/nbl/video/ILogicalDevice.h

```diff
@@ -232,7 +236,7 @@ class ILogicalDevice : public core::IReferenceCounted
         //! Utility wrapper for the pointer based func
         virtual void invalidateMappedMemoryRanges(core::SRange<const video::IDriverMemoryAllocation::MappedMemoryRange> ranges) = 0;
 
-        virtual core::smart_refctd_ptr<IGPUBuffer> createGPUBuffer(const IDriverMemoryBacked::SDriverMemoryRequirements& initialMreqs, const bool canModifySubData = false) { return nullptr; }
+        virtual core::smart_refctd_ptr<IGPUBuffer> createGPUBuffer(const IGPUBuffer::SCreationParams& creationParams) { return nullptr; }
```

Comment:
`canModifySubData` is needed for OpenGL, is it in the `SCreationParams` ?

P.S. keep track of it on Vulkan as well, and refuse (and log an error) an inline commandbuffer buffer update if the buffer doesn't have that capability set.

Reply:
No it isn't. I thought it wasn't required for Vulkan.

I'll add a member to `CVulkanBuffer`.

Reply:
resolved